RetsTalk/RetsTalk/Retrospect/Model/RetrospectManager.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RetrospectManager.swift
+//  RetsTalk
+//
+//  Created by KimMinSeok on 11/19/24.
+//
+
+import Foundation
+
+final class RetrospectManager: RetrospectManageable {
+    private(set) var retrospects: [Retrospect] = []
+    fileprivate var messageMap: [UUID: MessageManageable] = [:]
+    
+    func fetchRetrospects(offset: Int, mount: Int) {
+        
+    }
+    
+    func create() {
+        let retropsect = Retrospect(author: User(nickname: "alstjr"))
+        
+        let messageManager = MessageManager(
+            retrospectID: retropsect.id,
+            messageManagerListener: self
+        )
+        
+        retrospects.append(retropsect)
+        messageMap[retropsect.id] = messageManager
+    }
+    
+    func delete(_ retrospect: Retrospect) {
+        
+    }
+}
+
+extension RetrospectManager: MessageManagerListener {
+    func didFinishRetrospect(_ messageManager: MessageManager) {
+        guard let index = retrospects
+            .firstIndex(where: { $0.id == messageManager.retrospectID })
+        else { return }
+        retrospects[index].status = .finish
```

Comment:
```suggestion
        guard let index = retrospects.firstIndex { $0.id == messageManager.retrospectID }
        else { return }
        
        retrospects[index].status = .finish
```

firstIndex하나만 수행하고 길지 않은데 띄울 필요는 없는 것 같습니다.(trailing closure는 덤)

그리고 가드문 이후에 공백 !

아래도 마찬가지입니다.